mautic/mautic
# Fix SendGird Attachments on Queue Mode

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 4.x
| New feature/enhancement? (use the a.x branch)      | No
| Deprecations?                          | No
| BC breaks? (use the c.x branch)        |  No
| Automated tests included?              | Yes 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #10356 



#### Description:



#### Steps to test this PR:


1. Configure SendGrid to be your email sending service
2. Create an email and attachment to it
3. Try to send the email while you are using queue mode
4. When you receive the email there will be no attachment
5. Apply this PR
6. Re-test the same email you should get an attachment this time




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10566"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
